opentelemetry-sdk/tests/logs/test_handler.py

```diff
@@ -235,3 +239,59 @@ def test_log_record_trace_correlation(self):
             self.assertEqual(log_record.trace_id, span_context.trace_id)
             self.assertEqual(log_record.span_id, span_context.span_id)
             self.assertEqual(log_record.trace_flags, span_context.trace_flags)
+
+    def test_log_record_args_are_translated(self):
+        emitter_provider_mock = Mock(spec=LoggerProvider)
```

Comment:
Hi -- I've updated this file in main to use real types instead of mocks (PR [here](https://github.com/open-telemetry/opentelemetry-python/pull/4071)). Can you merge those changes in to this branch? Afterwards, you may want to use real types for these tests as well.

Reply:
Is it ok like this, or should `set_up_test_logging` stay without additional parameters?